modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json

```diff
@@ -134,47 +99,21 @@
       ]
     },
     {
-      "@id": "this:DigitalToAnalogConverterShape",
-      "@type": "sh:NodeShape",
-      "and": [
-        {
-          "node": "this:ConverterShape"
-        },
-        {
-          "property": [
-            {
-              "path": "nsg:scale",
-              "name": "Scale",
-              "datatype": "xsd:float",
-              "minCount": 1,
-              "maxCount": 1
-            }
-          ]
-        }
-      ]
-    },
-    {
-      "@id": "this:ConverterShape",
+      "@id": "this:MeasurementRepetitionShape",
```

Comment:
Rename to bbp/core/experiment/measurement/v0.1.0/shapes/RepeatedQuantitativeMeasurementShape so that it can be reused. It can also extends bbp/core/QuantitativeValue